### ISSUE TITLE
Git Flow button throws error

### ISSUE THREAD

**Issue by t-ricci-digit** [gitextensions/gitextensions#10406]
### Environment

- Git Extensions 4.0.0.15569
- Build cba315df5366fca01240d93b6e5a5077fd5e6630
- Git 2.38.1.windows.1
- Microsoft Windows NT 10.0.22621.0
- .NET 6.0.11
- DPI 96dpi (no scaling)

### Issue description

Exit code: 1
Command: C:\Program Files\Git\bin\git.exe
Arguments: config --get gitflow.branch.master
Working directory: C:\Users\tommaso.ricci\source\repos\ZTS-DigitalTwin\

```
GitExtUtils.ExternalOperationException
 ---> System.Exception
   --- End of inner exception stack trace ---
   at GitCommands.ExecutableExtensions.GetOutputAsync(IExecutable executable, ArgumentString arguments, Byte[] input, Encoding outputEncoding, CommandCache cache, Boolean stripAnsiEscapeCodes)
   at Microsoft.VisualStudio.Threading.JoinableTask.CompleteOnCurrentThread()
   at Microsoft.VisualStudio.Threading.JoinableTask`1.CompleteOnCurrentThread()
   at Microsoft.VisualStudio.Threading.JoinableTaskFactory.Run[T](Func`1 asyncMethod, JoinableTaskCreationOptions creationOptions)
   at Microsoft.VisualStudio.Threading.JoinableTaskFactory.Run[T](Func`1 asyncMethod)
   at GitCommands.ExecutableExtensions.GetOutput(IExecutable executable, ArgumentString arguments, Byte[] input, Encoding outputEncoding, CommandCache cache, Boolean stripAnsiEscapeCodes)
   at GitExtensions.Plugins.GitFlow.GitFlowForm.get_IsGitFlowInitialised()
   at GitExtensions.Plugins.GitFlow.GitFlowForm.Init()
   at GitExtensions.Plugins.GitFlow.GitFlowForm..ctor(GitUIEventArgs gitUiCommands)
   at GitExtensions.Plugins.GitFlow.GitFlowPlugin.Execute(GitUIEventArgs args)
   at GitUI.CommandsDialogs.FormBrowse.<>c__DisplayClass70_0.<RegisterPlugins>b__2(Object <p0>, EventArgs <p1>)
   at System.Windows.Forms.ToolStripMenuItem.OnClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   at System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.ToolStripDropDown.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   at System.Windows.Forms.ToolStrip.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)
```

Clicking Git Flow in Plugins gives the error below

### Steps to reproduce

Click the git flow button and see the error dialog

### Did this work in previous version of GitExtensions?

I only have this version and haven't tried before

**Comment by t-ricci-digit** [comment]
I tried installing Sourcetree and using git flow from there. I think Sourcetree installed and configured "git flow" and now it's working inside Git Extensions as well

**Comment by gerhardol** [comment]
Duplicate of #10334
Fix in for 4.0.1
Just init Git-flow from the command line (should be `git config gitflow.branch.master true`)
